subworkflows/nf-core/fastq_align_chromap/main.nf

```diff
@@ -0,0 +1,37 @@
+/*
+ * Map reads, sort, index BAM file and run samtools stats, flagstat and idxstats
+ */
+
+include { CHROMAP_CHROMAP         } from '../../../modules/nf-core/chromap/chromap/main'
+include { BAM_SORT_STATS_SAMTOOLS } from '../bam_sort_stats_samtools/main'
+
+workflow FASTQ_ALIGN_CHROMAP {
+    take:
+    reads // channel: [ val(meta), [ reads ] ]
+    index //    path: /path/to/index
+    fasta //    path: /path/to/fasta
```

Comment:
If I understood correctly, these should be prefixed with `ch_`

Reply:
Would be nice to clarify the guidelines since in many of the subworkflows currently is not the case e.g. https://github.com/JoseEspinosa/nf-core-modules/blob/2727a01bfab08f40f6f89ed7ed851d950e12cfed/subworkflows/nf-core/fastq_align_bowtie2/main.nf#L10-L14 or https://github.com/JoseEspinosa/nf-core-modules/blob/2727a01bfab08f40f6f89ed7ed851d950e12cfed/subworkflows/nf-core/gatk_tumor_normal_somatic_variant_calling/main.nf#L21-L30

Reply:
Addressed here 10dd532c63090e1f34ca2d67bf750dc25a145c60

Reply:
yes agreed, would have to go back and update older modules